strutil/strutil.go

```diff
@@ -70,3 +70,33 @@ func Quoted(names []string) string {
 
 	return strings.Join(quoted, ", ")
 }
+
+// WordWrap takes a input string and word wraps after `n` chars
+// into a new slice.
+//
+// Caveats:
+// - If a single word that is biger than max will not get wrapped
+// - Extra whitespace will be removed
+func WordWrap(s string, max int) []string {
+	n := 0
+
+	out := []string{}
+	line := bytes.NewBuffer(nil)
+	for _, word := range strings.Fields(s) {
+		if n+len(word) > max && n > 0 {
+			out = append(out, line.String())
+			line.Truncate(0)
+			n = 0
+		} else if n > 0 {
+			fmt.Fprintf(line, " ")
+			n += 1
+		}
+		fmt.Fprintf(line, word)
```

Comment:
The logic here is corrupting the spacing of the original line, which means indentation and tabling will be gone. A better approach would be finding the break point and then using the original content instead of rewriting it.